build

```diff
@@ -11,14 +11,22 @@ METEOR_DIR=$SCRIPTPATH/meteor_core
 
 [ -f $METEOR_DIR/client/loadClientBundle.html ] && rm $METEOR_DIR/client/loadClientBundle.html
 
-SERVER_BUNDLE="assets/server.bundle.js"
-CLIENT_BUNDLE="assets/client.bundle.js"
-
 cd $WEBPACK_DIR
-env NODE_ENV=production $SCRIPTPATH/node_modules/webpack/bin/webpack.js --config webpack.config.client.prod.js --progress --colors
-env NODE_ENV=production $SCRIPTPATH/node_modules/webpack/bin/webpack.js --config webpack.config.server.prod.js --progress --colors
-cd $METEOR_DIR
-ln -sf $WEBPACK_DIR/$SERVER_BUNDLE server/server.bundle.js
-ln -sf $WEBPACK_DIR/$CLIENT_BUNDLE client/client.bundle.js
-rm -rvf ../dist
-METEOR_SETTINGS=$(cat ../settings/prod.json) && meteor build ../dist
```

Comment:
We should keep the rest of this script the way it was and copy in only this line